recipes/libjpeg-turbo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.5.1" %}
+{% set libjpeg_version = "8c" %}
+
+package:
+    name: libjpeg-turbo
+    version: {{ version }}
+
+source:
+    fn: libjpeg-turbo-{{ version }}.tar.gz
+    url: http://downloads.sourceforge.net/project/libjpeg-turbo/{{ version }}/libjpeg-turbo-{{ version }}.tar.gz
+    md5: 55deb139b0cac3c8200b75d485fc13f3
+
+build:
+    number: 0
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+        
+requirements:
+    build:
+        - python  # [win]
+        - cmake  # [win]
+        - msinttypes  # [win]
+        - autoconf  # [unix]
+        - automake  # [unix]
+        - libtool  # [unix]
+        - yasm
+
+test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
+    files:
+        - testorig.jpg
+    commands:
+        - djpeg -dct int -ppm -outfile testout.ppm testorig.jpg
+
+about:
+    home: http://www.libjpeg-turbo.org/
+    license: Custom open source license + BSD
```

Comment:
Can you change this to: `libjpeg-turbo license and BSD x-Clause` add the BSD version in (2,  or 3) in the `x`?